.github/workflows/tg-frequent-tests-impl.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: tar -xvf ttm_${{ matrix.test-group.arch }}.tar
       - uses: ./.github/actions/install-python-deps
       - name: Run frequent regression tests
-        timeout-minutes: 60
+        timeout-minutes: 90
```

Comment:
are we hitting the timeout limit?

Reply:
Yeah we are, with the 2 additional resnet tests :(

Reply:
We should start splitting this into separate jobs by model/category similar to t3k freq now that we have more tests (not in this pr).

Reply:
please file a ticket for splitting into separate jobs

Reply:
Work for this is tracked here: https://github.com/tenstorrent/tt-metal/issues/13610